datastore/bolt/repo_test.go

```diff
@@ -1,24 +1,132 @@
 package bolt
 
 import (
+	"github.com/drone/drone/common"
+	"github.com/drone/drone/common/sshutil"
 	"testing"
-
-	. "github.com/franela/goblin"
+	//. "github.com/franela/goblin"
+	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestRepo(t *testing.T) {
-	g := Goblin(t)
+	/*g := Goblin(t)
 	g.Describe("Repos", func() {
 
-		g.It("Should find by name")
+		g.It("Should find by name") // ok
 		g.It("Should find params")
 		g.It("Should find keys")
-		g.It("Should delete")
-		g.It("Should insert")
-		g.It("Should not insert if exists")
+		g.It("Should delete") // ok
+		g.It("Should insert") // ok
+		g.It("Should not insert if exists")  // ?
 		g.It("Should insert params")
 		g.It("Should update params")
 		g.It("Should insert keys")
 		g.It("Should update keys")
 	})
+	*/
+	var testUser string = "octocat"
+	var testRepo string = "github.com/octopod/hq"
+	var db *DB //-- Temp database
+	testParamsIns := map[string]string{
+		"A": "Alpha",
+		"B": "Beta",
+		"C": "Charlie",
+		"D": "Delta",
+		"E": "Echo",
+	}
+	testParamsUpd := map[string]string{
+		"A": "Alpha-Upd",
+		"B": "Beta-Upd",
+		"C": "Charlie-Upd",
+		"D": "Delta-Upd",
+		"E": "Echo-Upd",
+	}
+
+	//--
+	db = Must("/tmp/drone.test.db")
```

Comment:
in the other unit tests we follow this pattern:

```
		// create a new database before each unit
		// test and destroy afterwards.
		g.BeforeEach(func() {
			db = Must("/tmp/drone.test.db")
		})
		g.AfterEach(func() {
			os.Remove(db.Path())
		})
```

this creates and destroys a database for each unit test, which provides better isolation and prevents unit tests from conflicting with eachother